python/tvm/relay/expr.py

```diff
@@ -98,39 +125,87 @@ def __init__(self,
 
 @register_relay_node
 class Call(Expr):
-    """A function call in Relay, see tvm/relay/expr.h for more details."""
+    """Function call node in Relay.
+
+    Call node corresponds the operator application node
+    in computational graph terminology.
+
+    Parameters
+    ----------
+    op: tvm.relay.Op or any tvm.relay.Expr with function type.
```

Comment:
`op: Union[tvm.relay.Op, tvm.relay.Expr]`

The operation to be called. If the operation is a `tvm.relay.Expr` it must be a function.

Reply:
better yet: why not tvm.relay.Expr?
Op has function type, so any tvm.relay.Expr with function type will work.